core/app/components/DataTable/PubsDataTable/PubsDataTableServer.tsx

```diff
@@ -0,0 +1,34 @@
+import type { CommunitiesId, UsersId } from "db/public";
+
+import { getPubsCount, getPubsWithRelatedValues } from "~/lib/server";
+import { PubsDataTable } from "./PubsDataTableClient";
+import { getFilterParamsFromSearch, searchParamsCache } from "./validations";
+
+type Props = {
+	communityId: CommunitiesId;
+	userId: UsersId;
+	searchParams: { [key: string]: string | string[] | undefined };
+};
+
+export const PubsDataTableServer = async ({ communityId, userId, searchParams }: Props) => {
```

Comment:
Not currently used anywhere, but should be ready to go when needed, with possible adjustments to `getColumns`.